api/file-translations/index.ts

```diff
@@ -123,10 +124,13 @@ export class SmartlingFileTranslationsApi extends SmartlingBaseApi {
         );
     }
 
-    async detectFileLanguage(accountUid: string, fileUid: string): Promise<LanguageDetectionDto> {
+    async detectFileLanguage(
+        accountUid: string, fileUid: string, params?: LanguageDetectionParameters
+    ): Promise<LanguageDetectionDto> {
         return await this.makeRequest(
             "post",
-            `${this.entrypoint}/${accountUid}/files/${fileUid}/language-detection`
+            `${this.entrypoint}/${accountUid}/files/${fileUid}/language-detection`,
+            JSON.stringify(params?.export())
```

Comment:
You have `Detect file language with empty parameters` test for `JSON.stringify({})` but for existing code (that doesn't pass `params` here) it will be `JSON.stringify(undefined)`. Will it break request in this case?

Reply:
tested 1. without parameters. 2. with empty parameters. it works.